setup

```diff
@@ -0,0 +1,210 @@
+    for file in so_files:
+        shutil.copy2(file, QUERY_MODULES_DIRECTORY)
+
+    #################################################
+    #             Load Python modules               #
+    #################################################
+
+    os.chdir(PY_DIRECTORY)
+    ignore_list = [
+        "tests",
+        "requirements.txt",
+        "pytest.ini",
+        "htmlcov",
+        "__pycache__",
+        ".*",
+    ]
+
+    # Copy python repository tree inside dist folder
+    copytree(PY_DIRECTORY, QUERY_MODULES_DIRECTORY, ignore_list)
+    os.environ["PYTHONPATH"] = PY_DIRECTORY
+
+    #################################################
+    #              Load Rust modules                #
+    #################################################
+
+    os.chdir(RS_DIRECTORY)
+    for project in filter(
+            lambda f: os.path.isdir(f) and f != "rsmgp-sys",
+            os.listdir(RS_DIRECTORY)):
+        project_dir = os.path.join(RS_DIRECTORY, project)
+        os.chdir(project_dir)
+        subprocess.run(["cargo", "build", "--release"])
+
+        release_dir = os.path.join(project_dir, "target", "release")
+        os.chdir(release_dir)
+        modules = glob.glob("*.so")
+        assert len(modules) == 1, f"Bad rust/{project} lib."
+
+        module = modules[0]
+        src_file = os.path.join(release_dir, module)
+        module = module.lstrip("lib") if module.startswith("lib") else module
+        dst_file = os.path.join(QUERY_MODULES_DIRECTORY, module)
+        shutil.copy2(src_file, dst_file)
+
+
+def run_build_action(copy_dist_dest_path: str):
+    log.log(level=log.INFO, msg="Starting building of source code...")
+    build()
+    log.log(level=log.INFO, msg="Building done.")
+
+    if copy_dist_dest_path is None:
+        log.log(level=log.INFO,
+                msg="Skipping copy of contents of 'dist' folder to specified folder since --path is not specified")
+        return
+
+    log.log(level=log.INFO,
+            msg='Copying files from folder {src} to {dst} folder.'.format(src=QUERY_MODULES_DIRECTORY,
+                                                                          dst=copy_dist_dest_path))
+
+    shutil.copytree(QUERY_MODULES_DIRECTORY, copy_dist_dest_path, dirs_exist_ok=True)
+    log.log(level=log.INFO, msg='Copying done!')
+
+
+def run_modules_storage_setup_action(modules_storage_path_param):
+    if modules_storage_path_param is None:
+        modules_storage_path = f"{pathlib.Path().resolve()}/dist"
+    else:
+        modules_storage_path = modules_storage_path_param
+
+    log.log(level=log.INFO,
+            msg="--query-modules-dir flag in {memgraph_conf} will be set to {path}".format(path=modules_storage_path,
+                                                                                       memgraph_conf=MG_CONF_PATH))
+
+    subprocess.run(['sed', '-E', '-i', "s@.*(--query-modules-directory).*@\\1={path}@".format(path=modules_storage_path), MG_CONF_PATH  ])
```

Comment:
Black magic
Is this changing the flag or appending to the previously set flag?

Reply:
This is completely changing it. 
What about black magic 😄 do you think this is not good or it can somehow be better?